lib/logcat.js

```diff
@@ -1,118 +1,80 @@
-"use strict";
+import { SubProcess } from 'teen_process';
+import log from './logger';
```

Comment:
do we want to do this? or maybe we just do:

```js
import { getLogger } from 'appium-logger';
const log = getLogger('Logcat');
```

that way we don't need to do `[LOGCAT STDERR]` below, it will just be blank for stdout and `[STDERR]` for stderr